tests/README.md

```diff
@@ -35,8 +35,11 @@ You can restrict mypy tests to a single version by passing `-p2` or `-p3.9`:
 
 ## pytype\_test.py
 
-This test requires Python 2.7 and Python 3.6. Pytype will
-find these automatically if they're in `PATH`.
+This test requires Python 2.7 and Python 3.6. Pytype will find these
```

Comment:
@rchen152 Is this still true? I think it just runs with the current Python version and will run fine with Python 3.8 or 3.9, correct?

Reply:
Ah yeah, that looks out of date. pytype_test should be able to run in any version pytype supports, which is currently 3.6-3.9.